x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -87,9 +87,9 @@ const ActionsComponent: React.FC<ActionProps> = ({
   );
   const eventType = getEventType(ecsData);
 
-  const isEventContextMenuEnabled = useMemo(
-    () => !!ecsData.event?.kind && ecsData.event?.kind[0] === 'event',
-    [ecsData.event?.kind]
+  const isEventContextMenuEnabledForEndpoint = useMemo(
```

Comment:
@kevinlog This will prevent users create event filters for agent.type !== endpoint. Let me know if that make sense to you

Reply:
This makes sense to me.  Can you test with some real data if you haven't already?  Just standing up an Agent/Endpoint with the Security integration and the System integration will get you Events from a variety of `agent.type`s.  You should be able to verify pretty easily.

Reply:
You mean test this condition? I was able to test it with real data. I had some agent.type === 'endpoint' and other agent.type === 'filebeat' and I was able to see that this option was disabled for `agent.type === 'filebeat'`. Is this what you were suggesting?

Reply:
@dasansol92 yep!  That sounds good.  If you still have the instance running, do you mind adding some screenshots to the description?